sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -73,20 +75,19 @@ protected void ConfigureTestEventHub(IHostBuilder builder)
                         {"AzureWebJobsStorage", StorageTestEnvironment.Instance.StorageConnectionString}
                     });
                 })
-                .ConfigureServices(services =>
-                {
-                    services.Configure<EventHubOptions>(options =>
-                    {
-                        options.CheckpointContainer = Guid.NewGuid().ToString("D").Substring(0, 13);
-                        // Speedup shutdown
-                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
-                    });
-                })
                 .ConfigureDefaultTestHost<T>(b =>
                 {
                     b.AddEventHubs(options =>
                     {
                         options.EventProcessorOptions.TrackLastEnqueuedEventProperties = true;
+                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
+                        if (initialOffsetOptions != null)
+                        {
+                            options.InitialOffsetOptions = initialOffsetOptions;
+                        }
+                        options.CheckpointContainer = _checkpointContainerName;
```

Comment:
Why didn't we need this before? We use checkpoints all over the place.

Reply:
We did use it before, but it was in the other config that has been removed. We also didn't need the containerName before.

Reply:
I see .
Do we need it as a field now?

Reply:
Yes..

Reply:
Nevermind, I removed the thing that was using it.